docs/source/project-management/sdc-sit-plan.rst

```diff
@@ -0,0 +1,63 @@
+.. _sdc-sit-plan:
+
+SDC Software Integration Test (SIT) Plan
+========================================
+
+The SDC test plan is based on the `SOC Test Plan <https://lasp.colorado.edu/galaxy/display/IMAP/IMAP+SOC+Test+Plan>`_, but makes the below customizations to help fit the needs of SDC specific testing. Planned SIT dates and planned functionality tested can also be found in the linked SOC Test Plan.
+
+Software Inegration Test Timeline
+---------------------------------
+====================== =============================================================================================================================
+Timeline               Step
+====================== =============================================================================================================================
+Start of development   SDC creates a SIT procedures page and adds/updates test steps throughout the development process
+~4 weeks prior to test SDC begins to finalize test steps
```

Comment:
I'm not sure if it would be helpful to link to the sections below within this table?